### ISSUE TITLE
Import API with Swagger and Config File, fails after upgrading to the latest version.

### ISSUE THREAD

**Issue by ramonpenteado** [Axway-API-Management-Plus/apim-cli#261]
**API-Mgr. CLI version**
> 1.10.0.

**API-Manager and Service-Pack Version**
> 7.7-20211130

**Expected behavior**
API imported

**Actual behavior**
Import ends with an error.

**Log-Output in Debug**
```
Import API :  Payment
/usr/bin/java

----------------------------------------------------------------------------------------
API-Manager CLI: 1.10.0
                                                                        
To report issues or get help, please visit: 
https://github.com/Axway-API-Management-Plus/apim-cli
----------------------------------------------------------------------------------------
Module: API - I M P O R T (1.10.0)
----------------------------------------------------------------------------------------
811    DEBUG nvironmentProperties| Trying to load environment properties from file: /opt/Axway/apimcli/apim-cli-1.10.0/conf/env.properties ... not found.
821    DEBUG nvironmentProperties| Loaded environment properties from file: /opt/Axway/apimcli/apim-cli-1.10.0/conf/env.dev.properties
1008   DEBUG    APIManagerAdapter| Logging in with User: 'apiadmin'
1747   DEBUG ManagerConfigAdapter| Load API-Manager configuration.
1850   INFO     APIManagerAdapter| Successfully connected to API-Manager (7.7.20211130) on: https://apimgr:8075
1970   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=API Regulated Org, id=null]
2010   DEBUG nagerPoliciesAdapter| Load policies with type: ROUTING from API-Manager
2024   DEBUG nagerPoliciesAdapter| Load policies with type: FAULT_HANDLER from API-Manager
2041   DEBUG nagerPoliciesAdapter| Load policies with type: REQUEST from API-Manager
2057   DEBUG nagerPoliciesAdapter| Load policies with type: RESPONSE from API-Manager
2080   INFO  SpecificationFactory| Reading API-Definition (Swagger/WSDL) from file: '/opt/Axway/API/Payment/Payment.json' (relative path)
2089   DEBUG SpecificationFactory| Handle API-Specification: '{
  "openapi": "3.0.0",
  "info": {
    "title": "API Payment Initiation - Open Banking Brasil",
    "description": "API de Inicia????o de Pagamentos, respons??vel por viabilizar as opera????es de ini......', apiDefinitionFile: '/opt/Axway/API/Payment/Payment.json'
2127   DEBUG SpecificationFactory| Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger2xSpecification
2138   DEBUG SpecificationFactory| Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger1xSpecification
2146   INFO  SpecificationFactory| Detected: Open API 3.0 specification. 
2164   INFO  IImportConfigAdapter| Loading image from: '/opt/Axway/API/Payment/image/images.png'
2168   DEBUG tomPropertiesAdapter| Read configured custom properties from API-Manager
2197   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=PublicOrg, id=null]
2224   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=Testing, id=null]
2245   INFO          APIImportApp| Lookup actual API based on Path: /open-banking/payments/v1, V-Host: mtls-api-proxy.dev-openbanking.demoaxway.com
2254   DEBUG APIManagerAPIAdapter| Sending request to find existing APIs: https://apimgr:8075/api/portal/v1.3/proxies?field=path&op=eq&value=%2Fopen-banking%2Fpayments%2Fv1
2349   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=null, id=c3e1d397-0074-4ade-acc4-d56660089b89]
2369   DEBUG APIManagerAPIAdapter| Found: 1 exposed API(s)
2376   DEBUG agerAPIMethodAdapter| Load API-Methods for API: d47a4e66-9bf3-4241-8067-3eb152ef0c9f from API-Manager.
2401   DEBUG IManagerQuotaAdapter| Load quotas with ID: 00000000-0000-0000-0000-000000000001 from API-Manager.
2429   DEBUG IManagerQuotaAdapter| Load quotas with ID: 00000000-0000-0000-0000-000000000000 from API-Manager.
2451   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=null, id=null]
2528   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: 5e6e70a0-2032-4735-8af3-acd39f39ef2b
2543   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: 4163a264-6501-49ed-913d-03aab1f4f69a
2556   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: c3e1d397-0074-4ade-acc4-d56660089b89
2569   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: 614af44c-39f4-493b-a2a7-7a5fe53e18f1
2581   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: 358f3b3d-4b4d-43f0-bb14-a602d176e74b
2595   DEBUG agerAPIAccessAdapter| Load API-Access with type: organizations from API-Manager with ID: 832ed790-ebb7-4af6-8edc-27f6c62336a0
2608   DEBUG    APIMgrAppsAdapter| Load subscribed applications for API-ID: d47a4e66-9bf3-4241-8067-3eb152ef0c9f from API-Manager
2647   DEBUG rOrganizationAdapter| Load organizations from API-Manager using filter: OrgFilter [name=null, id=614af44c-39f4-493b-a2a7-7a5fe53e18f1]
2665   INFO  APIManagerAPIAdapter| Loading application quotas for 3 subscribed applications.
2667   DEBUG IManagerQuotaAdapter| Load quotas with ID: 8bb09d80-ad01-43be-9a29-5c7247d79a77 from API-Manager.
2681   DEBUG IManagerQuotaAdapter| Load quotas with ID: 57099528-200d-4149-a825-a967d64afd38 from API-Manager.
2692   DEBUG IManagerQuotaAdapter| Load quotas with ID: d162136c-aabd-46e8-a2a6-5fa1ef902ed3 from API-Manager.
2737   DEBUG SpecificationFactory| Handle API-Specification: '{"openapi":"3.0.0","info":{"title":"API Payment Initiation - Open Banking Brasil","description":"API de Inicia????o de Pagamentos, respons??vel por viabilizar as opera????es de inicia????o de pagament......', apiDefinitionFile: 'pagamentos_2022-02-22T20:43:33Z.json'
2752   DEBUG SpecificationFactory| Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger2xSpecification
2757   DEBUG SpecificationFactory| Can't handle API specification with class: com.axway.apim.api.apiSpecification.Swagger1xSpecification
2825   INFO  APIManagerAPIAdapter| Found existing API: 'Pagamentos' (published) on path: '/open-banking/payments/v1' (ID: 'd47a4e66-9bf3-4241-8067-3eb152ef0c9f')
2828   DEBUG       APIChangeState| No change for property: apiDefinition[Desired: 'com.axway.apim.api.apiSpecification.OAS3xSpecification@6155d082' vs Actual: 'com.axway.apim.api.apiSpecification.OAS3xSpecification@3a5ecce3']
2828   DEBUG       APIChangeState| Ignoring Null-Property: caCerts[Desired: 'null' vs Actual: '[]']
2828   DEBUG       APIChangeState| No change for property: descriptionType[Desired: 'markdown' vs Actual: 'markdown']
2829   DEBUG       APIChangeState| No change for property: descriptionMarkdown[Desired: '${env.DOCUMENTS}/api-products/Payment/README_pt-BR.md' vs Actual: '${env.DOCUMENTS}/api-products/Payment/README_pt-BR.md']
2867   DEBUG       APIChangeState| No change for property: securityProfiles[Desired: '[SecurityProfile [[SecurityDevice [type=passThrough]]], SecurityProfile [[SecurityDevice [type=authPolicy], SecurityDevice [type=oauthExternal]]], SecurityProfile [[SecurityDevice [type=oauthExternal]]]]' vs Actual: '[SecurityProfile [[SecurityDevice [type=passThrough]]], SecurityProfile [[SecurityDevice [type=authPolicy], SecurityDevice [type=oauthExternal]]], SecurityProfile [[SecurityDevice [type=oauthExternal]]]]']
2868   DEBUG       APIChangeState| No change for property: authenticationProfiles[Desired: '[AuthenticationProfile [name=_default, isDefault=true, parameters={apiKey=pIxkyTpObtv3HKbwhWJ92NaixJL0pVJ3, apiKeyField=apikey, httpLocation=HEADER, password=}, type=apiKey]]' vs Actual: '[AuthenticationProfile [name=_default, isDefault=true, parameters={apiKey=pIxkyTpObtv3HKbwhWJ92NaixJL0pVJ3, apiKeyField=apikey, httpLocation=HEADER, password=}, type=apiKey]]']
2869   DEBUG       APIChangeState| No change for property: tags[Desired: '{Region=[Ljava.lang.String;@561868a0, Type=[Ljava.lang.String;@2ea6e30c, State=[Ljava.lang.String;@6138e79a, Role=[Ljava.lang.String;@2dcd168a, Standard=[Ljava.lang.String;@388526fb}' vs Actual: '{Role=[Ljava.lang.String;@21a21c64, Type=[Ljava.lang.String;@7803bfd, State=[Ljava.lang.String;@42bc14c1, Region=[Ljava.lang.String;@531f4093, Standard=[Ljava.lang.String;@62ef27a8}']
2872   DEBUG       APIChangeState| No change for property: outboundProfiles[Desired: '{_default=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default], paymentsGetConsentsConsentId=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPaymentConsent, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=null], paymentsPostConsents=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPaymentConsent, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=null], paymentsGetPixPaymentsPaymentId=OutboundProfile [routeType=policy, requestPolicy=ForwardConsentedAccounts, responsePolicy=null, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=null], paymentsPostPixPayments=OutboundProfile [routeType=policy, requestPolicy=POST-Payments-Initiation-Handler, responsePolicy=PaymentUpdateConsentStatus, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=null]}' vs Actual: '{_default=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default], paymentsGetConsentsConsentId=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPaymentConsent, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default], paymentsPostConsents=OutboundProfile [routeType=policy, requestPolicy=null, responsePolicy=null, routePolicy=routeToPaymentConsent, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default], paymentsGetPixPaymentsPaymentId=OutboundProfile [routeType=policy, requestPolicy=ForwardConsentedAccounts, responsePolicy=null, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default], paymentsPostPixPayments=OutboundProfile [routeType=policy, requestPolicy=POST-Payments-Initiation-Handler, responsePolicy=PaymentUpdateConsentStatus, routePolicy=routeToPayment, faultHandlerPolicy=EventLogFaultHandler, authenticationProfile=_default]}']
2872   DEBUG       APIChangeState| Ignoring Null-Property: serviceProfiles[Desired: 'null' vs Actual: '{_default=ServiceProfile [apiId=542fe981-de94-4b84-bec4-3643c791ee07, basePath=http://payments.open-banking-backend:8082]}']
2873   DEBUG       APIChangeState| No change for property: inboundProfiles[Desired: '{_default=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: OAuth External - Consent, corsProfile: _default, paymentsGetPixPaymentsPaymentId=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: Policy and Payments - Payments, corsProfile: null, paymentsPostPixPayments=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: Policy and Payments - Payments, corsProfile: null}' vs Actual: '{_default=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: OAuth External - Consent, corsProfile: _default, paymentsGetPixPaymentsPaymentId=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: Policy and Payments - Payments, corsProfile: null, paymentsPostPixPayments=monitorAPI: true, monitorSubject: authentication.subject.id, securityProfile: Policy and Payments - Payments, corsProfile: null}']
2873   DEBUG       APIChangeState| No change for property: corsProfiles[Desired: '[com.axway.apim.api.model.CorsProfile@6f3c660a]' vs Actual: '[com.axway.apim.api.model.CorsProfile@74f5ce22]']
2873   DEBUG       APIChangeState| Changed property: clientOrganizations[Desired: '['PublicOrg', 'API Regulated Org', 'Testing']' vs Actual: '['API Regulated Org', 'Testing']']
2874   DEBUG       APIChangeState| Ignoring Null-Property: applications[Desired: 'null' vs Actual: '[[Second TPP (8bb09d80-ad01-43be-9a29-5c7247d79a77)], [First TPP (57099528-200d-4149-a825-a967d64afd38)], [Postman (d162136c-aabd-46e8-a2a6-5fa1ef902ed3)]]']
2874   DEBUG       APIChangeState| No change for property: path[Desired: '/open-banking/payments/v1' vs Actual: '/open-banking/payments/v1']
2874   DEBUG       APIChangeState| No change for property: state[Desired: 'published' vs Actual: 'published']
2874   DEBUG       APIChangeState| No change for property: version[Desired: '1.0.0' vs Actual: '1.0.0']
2875   DEBUG       APIChangeState| No change for property: vhost[Desired: 'mtls-api-proxy.dev-openbanking.demoaxway.com' vs Actual: 'mtls-api-proxy.dev-openbanking.demoaxway.com']
2875   DEBUG       APIChangeState| No change for property: name[Desired: 'Pagamentos' vs Actual: 'Pagamentos']
2876   DEBUG       APIChangeState| No change for property: summary[Desired: 'O PIX ? o sistema criado para efetuar pagamentos instant?neos, A API permite transfer?ncias e pagamentos em tempo real.' vs Actual: 'O PIX ? o sistema criado para efetuar pagamentos instant?neos, A API permite transfer?ncias e pagamentos em tempo real.']
2971   DEBUG       APIChangeState| No change for property: image[Desired: 'APIImage [bytes=4497]' vs Actual: 'APIImage [bytes=6062]']
2972   DEBUG       APIChangeState| Changed property: systemQuota[Desired: 'APIQuota [id=null, type=SYSTEM, restrictions=[QuotaRestriction [api=*, method=paymentsGetPixPaymentsPaymentId, type=throttle, config={period=second, per=1, messages=300}], QuotaRestriction [api=*, method=paymentsGetConsentsConsentId, type=throttle, config={period=second, per=1, messages=300}]]]' vs Actual: 'APIQuota [id=null, type=SYSTEM, restrictions=[QuotaRestriction [api=d47a4e66-9bf3-4241-8067-3eb152ef0c9f, method=67d4f54c-72b9-4b47-bb7e-d54441912d4d, type=throttle, config={period=second, per=1, messages=300}]]]']
2973   DEBUG       APIChangeState| No change for property: organization[Desired: ''API Regulated Org'' vs Actual: ''API Regulated Org'']
2973   DEBUG       APIChangeState| Ignoring Null-Property: retirementDate[Desired: 'null' vs Actual: '0']
2974   INFO      APIImportManager| Recognized the following changes. Potentially Breaking: [] plus Non-Breaking: [clientOrganizations, systemQuota]
2976   INFO      APIImportManager| Update API Strategy: All changes can be applied in current state.
2976   DEBUG     APIImportManager| Apply breaking changes: [] & and Non-Breaking: [clientOrganizations, systemQuota], for PUBLISHED
2978   INFO     UpdateExistingAPI| Update existing published API: 'Pagamentos' 1.0.0 (ID: d47a4e66-9bf3-4241-8067-3eb152ef0c9f)
2978   DEBUG       APIChangeState| API-Proxy requires no updates
2981   INFO       APIQuotaManager| Updating System default quota for API: Pagamentos
2981   DEBUG      APIQuotaManager| System default-Restrictions: Desired: '[QuotaRestriction [api=*, method=paymentsGetPixPaymentsPaymentId, type=throttle, config={period=second, per=1, messages=300}], QuotaRestriction [api=*, method=paymentsGetConsentsConsentId, type=throttle, config={period=second, per=1, messages=300}]]', Actual: '[QuotaRestriction [api=d47a4e66-9bf3-4241-8067-3eb152ef0c9f, method=67d4f54c-72b9-4b47-bb7e-d54441912d4d, type=throttle, config={period=second, per=1, messages=300}]]'
2983   WARN  agerAPIMethodAdapter| No operation found with ID: 'paymentsGetPixPaymentsPaymentId' for API: 'd47a4e66-9bf3-4241-8067-3eb152ef0c9f'
2984   ERROR         APIImportApp| null
java.lang.NullPointerException: null
        at com.axway.apim.apiimport.actions.APIQuotaManager.updateRestrictions(APIQuotaManager.java:74) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.actions.APIQuotaManager.execute(APIQuotaManager.java:47) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.actions.UpdateExistingAPI.execute(UpdateExistingAPI.java:60) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.APIImportManager.applyChanges(APIImportManager.java:73) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:103) [apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:58) [apimcli-apis-1.10.0.jar:1.10.0]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_322]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_322]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_322]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_322]
        at com.axway.apim.cli.APIManagerCLI.run(APIManagerCLI.java:141) [apimcli-tool-1.10.0.jar:1.10.0]
        at com.axway.apim.cli.APIManagerCLI.main(APIManagerCLI.java:77) [apimcli-tool-1.10.0.jar:1.10.0]
2993   DEBUG    APIManagerAdapter| Closing cache ...
[ERROR] Unable to IMPORT api Payment !
```

**Test case sample**
> by running apim.sh api import -s ${ENV} -c ${rep}/$api-${ENV}-config.json -a ${rep}/${api}.json -force

**Important Note**
The same file perfectly works with earlier versions.

**Comment by cwiechmann** [comment]
Hi @ramonpenteado, Can you please share the API-Config and API-Specification files you used for the import with me. If you cannot share it here public, please send it by mail. Thanks.

**Comment by cwiechmann** [comment]
Thanks for sharing the configuration & API-Spec @ramonpenteado. With that config you provided I had no problem to import this API into my API-Manager:
```
C:\temp\payment-api>apim api import -s demo -c Payment-dev-config.json -a Payment.json
----------------------------------------------------------------------------------------
API-Manager CLI: 1.10.0

To report issues or get help, please visit:
https://github.com/Axway-API-Management-Plus/apim-cli
----------------------------------------------------------------------------------------
Module: API - I M P O R T (1.10.0)
----------------------------------------------------------------------------------------
1890   INFO     APIManagerAdapter| Successfully connected to API-Manager (7.7.20211130) on: https://manager.axway-apim-on-eks.de:443
2272   INFO  SpecificationFactory| Reading API-Definition (Swagger/WSDL) from file: 'Payment.json' (relative path)
2306   INFO  SpecificationFactory| Detected: Open API 3.0 specification.
2529   INFO  rOrganizationAdapter| No organization found using filter: OrgFilter [name=PublicOrg, id=null]
2530   WARN  IImportConfigAdapter| Unknown organization with name: 'PublicOrg' configured. Ignoring this organization.
2598   INFO  rOrganizationAdapter| No organization found using filter: OrgFilter [name=API NonRegulated Org, id=null]
2598   WARN  IImportConfigAdapter| Unknown organization with name: 'API NonRegulated Org' configured. Ignoring this organization.
2665   INFO  rOrganizationAdapter| No organization found using filter: OrgFilter [name=API Regulated Org, id=null]
2666   WARN  IImportConfigAdapter| Unknown organization with name: 'API Regulated Org' configured. Ignoring this organization.
2804   INFO  rOrganizationAdapter| No organization found using filter: OrgFilter [name=Testing, id=null]
2804   WARN  IImportConfigAdapter| Unknown organization with name: 'Testing' configured. Ignoring this organization.
2806   INFO          APIImportApp| Lookup actual API based on Path: /open-banking/payments/v1, V-Host: openbanking-dev-apim.demo.axway.com:8083
4107   INFO  APIManagerAPIAdapter| Found existing API: 'Pagamentos' (published) on path: '/open-banking/payments/v1' (ID: 'f1e64dc5-9dae-4360-946d-f64c401b7872')
4274   INFO      APIImportManager| Recognized the following changes. Potentially Breaking: [] plus Non-Breaking: [summary]
4274   INFO      APIImportManager| Update API Strategy: Re-Publish API to apply changes
4276   INFO  RepublishToUpdateAPI| Unpublish existing API for update: 'Pagamentos' (ID: f1e64dc5-9dae-4360-946d-f64c401b7872)
4377   INFO     UpdateExistingAPI| Update existing unpublished API: 'Pagamentos' 1.0.0 (ID: f1e64dc5-9dae-4360-946d-f64c401b7872)
4378   INFO        APIChangeState| Updating Frontend-API (Proxy) for the following properties: summary
4875   INFO       APIQuotaManager| Updating System default quota for API: Pagamentos
5569   INFO      ManageClientApps| All desired applications: [] have already a subscription. Nothing to do.
5570   INFO     UpdateExistingAPI| Successfully updated published API: 'Pagamentos' 1.0.0 (ID: f1e64dc5-9dae-4360-946d-f64c401b7872)
```

### Quotas are the problem here

But there must be a difference between the config-file you shared and the one that is producing this error. As we already discussed via Teams, the Log-Information tells me there are System-Quotas defined for this API, which is not the case in the config you gave me:
`
Changed property: systemQuota[Desired: 'APIQuota [id=null, type=SYSTEM, restrictions=[QuotaRestriction [api=*, method=paymentsGetPixPaymentsPaymentId, type=throttle, config={period=second, per=1, messages=300}], QuotaRestriction [api=*, method=paymentsGetConsentsConsentId, type=throttle, config={period=second, per=1, messages=300}]]]' vs Actual: 'APIQuota [id=null, type=SYSTEM, restrictions=[QuotaRestriction [api=d47a4e66-9bf3-4241-8067-3eb152ef0c9f, method=67d4f54c-72b9-4b47-bb7e-d54441912d4d, type=throttle, config={period=second, per=1, messages=300}]]]']
`

I have added some systemQuotas in the same way:

```json
   "systemQuota":{
      "restrictions":[
         {
            "method":"paymentsGetConsentsConsentId",
            "type":"throttle",
            "config":{
               "period":"minute",
               "messages":1000,
               "per":5
            }
         }
      ]
   }
```

 and with that I see the same error:
```
4630   WARN  agerAPIMethodAdapter| No operation found with ID: 'paymentsGetConsentsConsentId' for API: 'f1e64dc5-9dae-4360-946d-f64c401b7872'
4631   ERROR         APIImportApp| null
java.lang.NullPointerException: null
        at com.axway.apim.apiimport.actions.APIQuotaManager.updateRestrictions(APIQuotaManager.java:74) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.actions.APIQuotaManager.execute(APIQuotaManager.java:47) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.actions.UpdateExistingAPI.execute(UpdateExistingAPI.java:60) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.apiimport.APIImportManager.applyChanges(APIImportManager.java:73) ~[apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:103) [apimcli-apis-1.10.0.jar:1.10.0]
        at com.axway.apim.APIImportApp.importAPI(APIImportApp.java:58) [apimcli-apis-1.10.0.jar:1.10.0]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:564) ~[?:?]
        at com.axway.apim.cli.APIManagerCLI.run(APIManagerCLI.java:141) [apimcli-tool-1.10.0.jar:1.10.0]
        at com.axway.apim.cli.APIManagerCLI.main(APIManagerCLI.java:77) [apimcli-tool-1.10.0.jar:1.10.0]
```

### Workarounds for now

There are three possible workarounds until this is fully fixed:
- don't define a quota on method level, as it is causing this problem
- or use the toggle `-quotaMode ignore`
- or entirely remove the quota

**Comment by cwiechmann** [comment]
Hi @ramonpenteado, the attached snapshot version contains a fix for this problem. 
[axway-apimcli-1.10.1-SNAPSHOT.zip](https://github.com/Axway-API-Management-Plus/apim-cli/files/8131686/axway-apimcli-1.10.1-SNAPSHOT.zip)

Can you please try with that one and let me know, if it works on your side? If you have confirmed, I will create the release 1.10.1. Thanks!
